pom.xml

```diff
@@ -1838,7 +1838,7 @@
         <groupId>org.codehaus.jackson</groupId>
         <artifactId>jackson-mapper-asl</artifactId>
         <version>${codehaus.jackson.version}</version>
-        <scope>test</scope>
+        <scope>${hive.deps.scope}</scope>
```

Comment:
Hadoop 3.3.5+ do not need this: https://issues.apache.org/jira/browse/HADOOP-13332